sesame/src/main/scala/org/w3/banana/sesame/Sesame.scala

```diff
@@ -11,6 +11,7 @@ trait Sesame extends RDF {
   // types related to the RDF datamodel
   type Graph = Model
   type Triple = Statement
+  type Quad = Statement
```

Comment:
ok, one good thing about having `Quad` is that it makes clear that Sesame's Statement is actually a Quad.